docs/flatpak.md

```diff
@@ -1,110 +1,76 @@
 ## Building Flatpaks
 
-In addition to building docker images from Dockerfiles, atomic-reactor can also build Flatpak OCI images for both runtimes and applications. A Flatpak OCI image is defined by two things: by a flatpak.json file that contains metadata data about how to build the Flaptak, and a [module](https://docs.pagure.org/modularity/docs.html) containing the RPMs to build into the Flatpak.
-
-### Example build.json
-
-```json
-{
-    "source": {
-        "provider": "git",
-        "uri": "git://pkgs.fedoraproject.org/modules/flatpak-runtime"
-    },
-    "image": "org.fedoraproject.platform",
-    "prebuild_plugins": [
-        { "name": "flatpak_create_dockerfile",
-          "args": {
-              "module_name": "flatpak-runtime",
-              "module_stream": "f26",
-              "odcs_url": "https://odcs.fedoraproject.org/odcs/1",
-              "odcs_insecure": false,
-              "pdc_url": "https://pdc.fedoraproject.org/rest_api/v1",
-              "pdc_insecure": false,
-          }
-        },
-        { "name": "flatpak_create_dockerfile",
-          "args": {
-              "base_image": "registry.fedoraproject.org/fedora:26"
-          }
-        },
-        { "name": "pull_base_image",
-          "args": {
-              "parent_registry": "registry.fedoraproject.org"
-          }
-        },
-        { "name": "inject_yum_repo",
-          "args": {}
-        }
-    ],
-    "prepublish_plugins": [
-        { "name": "flatpak_create_oci" }
-    ],
-    "postbuild_plugins": [
-        { "name": "tag_and_push",
-                "args": {
-                        "registries":
-                                { "localhost:5000":
-                                  "insecure": true
-                                }
-                        }
-        }
-    ]
-}
+In addition to building docker images from Dockerfiles, atomic-reactor can also build Flatpak OCI images for both runtimes and applications. A Flatpak OCI image is defined by two things: by a [module](https://docs.pagure.org/modularity/docs.html) containing the RPMs to build into the Flatpak, and by information included in a `container.yaml` file.
 
-```
+Building Flatpaks requires, in addition to a Koji installation:
 
-### Dependencies
+ * A [MBS (module-build-service)](https://pagure.io/fm-orchestrator/) instance set up to build modules in Koji
+ * A [PDC (product definition center)](https://github.com/product-definition-center/product-definition-center) instance to store information about the built modules
+ * An [ODCS (on demand compose service)](https://pagure.io/odcs/) instance to create repositories for the build modules
 
-To build flatpaks that actually run currently requires:
+A modified version of osbs-box that sets up everything other than the MBS instance and contains scripts to create fake modules can be found at:
```

Comment:
Integrating with the main osbs-box would be nice, and there's nothing fundamentally incompatible - it does require me to find a few days, since my branch is a mess of:

 - Straightforward Flatpak additions
 - Tweaking around the configuration/hostsnames, etc, in order to make things work
 - Semi-successful attempts to make 'osbs-box up --no-cleanup' work better
 - Updates for recent openshift, OSBS that duplicate/conflict with things done in the official osbs-box

I'll try to find time at some point to do it ... but it might not be soon.

I'll update the README to point to 
   https://github.com/owtaylor/minimal-runtime
   https://github.com/owtaylor/banner

Which make up the "hello world" that my osbs-box fork uses - pretty literally - see https://github.com/owtaylor/osbs-box#building-flatpaks